aaelf32/aaelf32.rst

```diff
@@ -0,0 +1,3289 @@
+  | 48   | :code:`R_ARM_THM_MOVT_ABS`      | No       | MOVT                                     | X & 0xFFFF0000      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 49   | :code:`R_ARM_THM_MOVW_PREL_NC`  | No       | MOVW                                     | X & 0x0000FFFF      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 50   | :code:`R_ARM_THM_MOVT_PREL`     | No       | MOVT                                     | X & 0xFFFF0000      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 51   | :code:`R_ARM_THM_JUMP19`        | Yes      | B<cond>.W                                | X & 0x001FFFFE      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 53   | :code:`R_ARM_THM_ALU_PREL_11_0` | Yes      | ADR.W                                    | X & 0x00000FFF      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 54   | :code:`R_ARM_THM_PC12`          | Yes      | LDR<,B,SB,H,SH> (literal)                | ABS(X) & 0x00000FFF |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 87   | :code:`R_ARM_THM_MOVW_BREL_NC`  | No       | MOVW                                     | X & 0x0000FFFF      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 88   | :code:`R_ARM_THM_MOVT_BREL`     | No       | MOVT                                     | X & 0xFFFF0000      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 89   | :code:`R_ARM_THM_MOVW_BREL`     | Yes      | MOVW                                     | X & 0x0000FFFF      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+  | 131  | :code:`R_ARM_THM_GOT_BREL12`    | Yes      | | LDR(immediate, Thumb) 12-bit immediate | X & 0x00000FFF      |
+  +------+---------------------------------+----------+------------------------------------------+---------------------+
+
+The formation of the initial addend in a REL type relocation for the various
+instruction classes is described in the following table.  Insn modification describes how the
+result X is written back to the instruction; Result_Mask is the value of X
+after the masking operation described in `Static Thumb-16 relocations`_
+or `Static Thumb-32 instruction relocations`_ has been applied.
+
+.. _Thumb relocation actions by instruction type:
+
+.. table:: Thumb relocation actions by instruction type
+  :class: longtable
```

Comment:
as described above on the version history table, please use a class to specify the width and no longtable